mipengine/controller/cleaner.py

```diff
@@ -147,136 +209,91 @@ def _get_node_info_by_id(self, node_id: str) -> _NodeInfoDTO:
         )
 
     # This is only supposed to be called from a test.
-    # In all other circumstances cleanup should not be reset manually
-    def _reset_cleanup(self):
-        self._cleanup_file_processor._initialize_cleanup_file()
+    # In all other circumstances the cleaner should not be reset manually
+    def _reset(self):
+        self._cleanup_files_processor._delete_all_entries()
 
 
-def _create_node_task_handler(node_info: _NodeInfoDTO) -> NodeAlgorithmTasksHandler:
+def _get_node_task_handler(node_info: _NodeInfoDTO) -> NodeAlgorithmTasksHandler:
     return NodeAlgorithmTasksHandler(
         node_id=node_info.node_id,
         node_queue_addr=node_info.queue_address,
         node_db_addr=node_info.db_address,
-        tasks_timeout=node_info.tasks_timeout,
+        tasks_timeout=2,  # node_info.tasks_timeout,
```

Comment:
Forgotten.

Reply:
yes I forgot that. I have a `celery_cleanup_task_timeout` entry in `.deployment.toml` and it read from there